src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -1133,4 +1133,5 @@ bool NonClientIslandWindow::_IsTitlebarVisible() const
 void NonClientIslandWindow::SetTitlebarBackground(winrt::Windows::UI::Xaml::Media::Brush brush)
 {
     _titlebar.Background(brush);
+    _titlebar.SetBackgroundBrush(brush);

```

Comment:
Should `.Background()` just be inside `SetBackgroundBrush`? Also, there's now _two setters for the background brush :/_

Reply:
1. maybe
2. gods it's horrible: cause TitlebarControl is a Grid, it has a Background method, but I can't override that, so 🤷

Reply:
Okay weird question . . . can you add a listener for `PropertyChanged` entirely inside TitleBarControl? It listens to itself for Background changes

Reply:
And then you don't need to add another setter OR call a different setter :O